docs/chapters/getting-started/development-workflow.rst

```diff
@@ -0,0 +1,18 @@
+Development Workflow
+====================
+
+The development workflow depends on what is intended to be developed. If the intention is to add or modify a component/service then the SDK is the main track. If it's a bigger development task modifying the platform itself then the more appropriate track is to build the system image from sources.
```

Comment:
If the intention is to add or modify a component/service then the SDK is the main track can be rephrased to "If the intention is to add or modify a component/service then one should use the SDK".
The last sentence we can replace track with "way"